test/tests/functional/pbs_cray_suspend_resume.py

```diff
@@ -0,0 +1,93 @@
+# coding: utf-8
+
+# Copyright (C) 1994-2016 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+# details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# The PBS Pro software is licensed under the terms of the GNU Affero General
+# Public License agreement ("AGPL"), except where a separate commercial license
+# agreement for PBS Pro version 14 or later has been executed in writing with
+# Altair.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software - under
+# a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+import time
+from tests.functional import *
+
+
+@tags('cray')
+class TestSuspendResumeOnCray(TestFunctional):
+    """
+    Test special cases where suspend/resume functionality differs on cray
+    as compared to other platforms.
+    This test suite expects the platform to be 'cray' and assumes that
+    suspend/resume feature is enabled on it.
+    """
+    def setUp(self):
+        if not self.du.get_platform().startswith('cray'):
```

Comment:
Sorry about a late reply
While this is true that platform types can have something else starting with 'cray' in them, I see the likelihood of this happening to be rare and even if it happens it will be very rare that the test will fail because it will be some kind of 'cray' platform.
I also feel that a change in get_platform() from PTL will be published and we can use that information to decide whether or not the test needs a change.
what do you think?

Reply:
Thanks for the response.  That makes sense to me.  I am fine with it.